app-cdr/uif2iso/files/Makefile

```diff
@@ -0,0 +1,5 @@
+OBJS = $(patsubst %.c,%.o,$(wildcard *.c))
+CPPFLAGS += -DMAGICISO_IS_SHIT
```

Comment:
To be honest, I don't think this change is really worth adding another Makefile. For this specific package, CFLAGS and CPPFLAGS are used in the same place, so it doesn't change anything.